docs/custom_tool_guides/tool_guide.md

```diff
@@ -203,4 +203,4 @@ curl --location 'http://localhost:8000/chat-stream' \
 
 ## Step 6 (extra): Add Unit tests
 
-If you would like to go above and beyond, it would be helpful to add some unit tests to ensure that your tool is working as expected. Create a file [here](https://github.com/cohere-ai/toolkit/tree/main/src/backend/tests/tools) and add a few cases.
\ No newline at end of file
+If you would like to go above and beyond, it would be helpful to add some unit tests to ensure that your tool is working as expected. Create a file [here](https://github.com/cohere-ai/toolkit/tree/main/src/backend/tests/tools) and add a few cases.
```

Comment:
It seems like this one was not updated 😅 Other than that, LGTM!

Reply:
Ah my bad - fixed it!